WrfTrigger.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/python3
+
+# Download Rain cell files, download Mean-ref files.
+# Check Rain fall files are available in the bucket.
+
+import fnmatch
+import sys
+import os
+import json
+import getopt
+import datetime
+from google.cloud import storage
+from curw.rainfall.wrf.extraction import observation_utils as utils
+from curwmysqladapter import MySQLAdapter, Data
+
+
+def usage():
+    usage_text = """
+Usage: ./CSVTODAT.py [-d YYYY-MM-DD] [-t HH:MM:SS] [-h]
+
+-h  --help          Show usage
+-d  --date          Date in YYYY-MM-DD. 
+-t  --time          Time in HH:00:00.
+-f  --forward       Future day count
+-b  --backward      Past day count
+-T  --tag           Tag to differential simultaneous Forecast Runs E.g. wrf1, wrf2 ...
+    --wrf-rf        Path of WRF Rf(Rainfall) Directory. Otherwise using the `RF_DIR_PATH` from CONFIG.json
+    --wrf-kub       Path of WRF kelani-upper-basin(KUB) Directory. Otherwise using the `KUB_DIR_PATH` from CONFIG.json
+"""
+    print(usage_text)
+
+def download_wrf_files():
+    try:
+        client = storage.Client.from_service_account_json(key_file)
+        bucket = client.get_bucket(bucket_name)
+        prefix = initial_path_prefix + '_'
+        blobs = bucket.list_blobs(prefix=prefix)
+        print("prefix : ", prefix)
+        print("net_cdf_file_name : ", net_cdf_file_name)
+        for blob in blobs:
+            if fnmatch.fnmatch(blob.name, "*" + net_cdf_file_name):
+                print(blob.name)
+                directory = download_location + "/" + net_cdf_file_name
+                if not os.path.exists(download_location):
+                    os.makedirs(download_location)
+                blob.download_to_filename(directory)
+                download_raincell_file(directory)
+                return True
+        return False
+    except:
+        print('Rain cell/Mean-Ref/Rain fall file download failed')
+        return False
+
+
+def download_raincell_file(net_cdf_file_name):
+    adapter = MySQLAdapter(host=MYSQL_HOST, user=MYSQL_USER, password=MYSQL_PASSWORD, db=MYSQL_DB)
+    #start_ts_lk = '2018-05-24_08:00'
+    obs_stations = {'Kottawa North Dharmapala School': [79.95818, 6.865576, 'A&T Labs', 'wrf_79.957123_6.859688'],
```

Comment:
Can't understand the use of this config. But better to move as a seperate config file. May be inside of `META_FLO2D`